README.md

```diff
@@ -52,22 +59,54 @@ To start the Clearwater services, run:
 
 #### Scaling the deployment
 
-Having started up a deployment, it is then possible to scale it by adding more Sprout, Memcached, Chronos or Cassandra nodes.   E.g. to spin up an additional node of each of these types, run:
+Having started up a deployment, it is then possible to scale it by adding more Sprout, Astaire, Chronos or Cassandra nodes.   E.g. to spin up an additional node of each of these types, run:
 
-    sudo docker-compose -f minimal-distributed.yaml scale sprout=2 memcached=2 chronos=2 cassandra=2
+    sudo docker-compose -f minimal-distributed.yaml scale sprout=2 astaire=2 chronos=2 cassandra=2
 
 Note that scaling of Docker deployments is a work in progress and there are currently a number of known issues...
 
 * Bono doesn’t automatically start using new Sprout nodes unless the Bono process is restarted, e.g.
 
     `sudo docker exec clearwaterdocker_bono_1 sudo service bono restart`
-    
+
 * Homestead-prov and Ellis don’t load balance across multiple Cassandra nodes.
 
 * There is no tested or documented process for scaling down clusters of storage nodes in Docker -- it isn't sufficient to just delete the containers as they need to be explicitly decommissioned and removed from the clusters.
 
 If you scale up the clusters of storage nodes, you can monitor progress as new nodes join the clusters by running `utils/show_cluster_state.sh`.
 
+## Using Kubernetes
+
+Instead of using Docker Compose, you can deploy Clearwater in Kubernetes. This
+requires a Kubernetes cluster, and a Docker repository.
+
+### Prepare the images and edit the yaml files
+
+- First, build all the required images locally.
+
+        # Build the Clearwater docker images.
+        cd clearwater-docker
+        for i in base astaire cassandra chronos bono ellis homer homestead ralf sprout ; do docker build -t clearwater/$i $i ; done
```

Comment:
After this step, I noticed there's no `/etc/cassandra/cassandra.yaml ` in the cassandra image, is it expected?